src/mvba/mvba/mod.rs

```diff
@@ -223,19 +229,23 @@ impl Mvba {
         Ok(())
     }
 
-    // TODO: make me better, no unwrap?
-    fn must_get_proposer_votes(&mut self, proposer: &NodeId) -> &mut HashMap<NodeId, Vote> {
+    fn must_get_proposer_votes(&mut self, proposer: &NodeId) -> Result<&mut HashMap<NodeId, Vote>> {
```

Comment:
Should the `must` in this function name be `mut`? How about renaming to:
```suggestion
    fn get_proposer_votes_mut(&mut self, proposer: &NodeId) -> Result<&mut HashMap<NodeId, Vote>> {
```